src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1599,6 +1612,11 @@ private function getCommitMessageFromUser() {
         $commit = head($this->getRepositoryAPI()->getLocalCommitInformation());
         $template = $commit['message'];
       } else {
+        $revert_plan_check_paths = $this->getConfigurationManager()->getConfigFromAnySource(
```

Comment:
nit: maybe move this to a function whcih returns whether revert plan check paths is enabled.